src/Microsoft.AspNet.Mvc.Core/Rendering/Html/HtmlHelper.cs

```diff
@@ -494,7 +494,15 @@ public Task RenderPartialAsync([NotNull] string partialViewName, object model, V
             var view = viewEngineResult.View;
             using (view as IDisposable)
             {
-                var viewContext = new ViewContext(ViewContext, view, newViewData, TempData, writer);
+                var htmlHelperOptions = new HtmlHelperOptions
+                {
+                    ClientValidationEnabled = ViewContext.ClientValidationEnabled,
+                    Html5DateRenderingMode = ViewContext.Html5DateRenderingMode,
+                    ValidationMessageElement = ViewContext.ValidationMessageElement,
+                    ValidationSummaryMessageElement = ViewContext.ValidationSummaryMessageElement
+                };
+
+                var viewContext = new ViewContext(ViewContext, view, newViewData, TempData, writer, htmlHelperOptions);
```

Comment:
the `ViewContext` copy constructor should not also require an `HtmlHelperOptions` argument. copy what is needed from the original `ViewContext` in the copy constructor.